code/modules/mob/transform_procs.dm

```diff
@@ -481,8 +481,6 @@
 	regenerate_icons()
 	icon = null
 	invisibility = INVISIBILITY_MAXIMUM
-	for(var/t in bodyparts)
-		qdel(t)
```

Comment:
All the transformation procs qdel the mob at the end, which qdels all bodyparts anyways

It also interferes with robotize because doing this also qdeletes the brain, which we need to transfer into the cyborg's MMI